apps-api/backoffice/src/utils/error.ts

```diff
@@ -119,3 +130,68 @@ export function err<E>(_err: E | Err<never, E>): Err<never, E> {
 
   return defaultErr(errBody) as Err<never, E>
 }
+
+export const throwWithReturnType = <E>(err: E): E => {
+  throw err
+}
```

Comment:
what is the purpose of this kub?
normally we don't return the throw
and we return expected error as err()